### PR TITLE
OBPIH-3242 Add source name to PO lines export and import

### PR DESCRIPTION
**Pull request by DominikaHatala**

### CODE REVIEW

**Review by awalkowiak — APPROVED** [review]

File: grails-app/services/org/pih/warehouse/data/ProductSupplierDataService.groovy [hunk 1]
```diff
@@ -133,15 +133,15 @@ class ProductSupplierDataService {
...
+            return ProductSupplier.get(params.productSupplier)
         }
 
         Product product = Product.get(params.product.id)
         ProductSupplier productSupplier = new ProductSupplier()
-        productSupplier.code = params.productSupplier.id?:params.supplierCode
+        productSupplier.code = params.productSupplier?:params.supplierCode
         productSupplier.supplierCode = params.supplierCode
-        productSupplier.name = params.supplierCode
+        productSupplier.name = params.sourceName ?: params.supplierCode
```
Comment (jmiranda) [inline grails-app/services/org/pih/warehouse/data/ProductSupplierDataService.groovy:144]: I think we should ask kelsey about this, but I think this should probably be one of two options:

Option A
```
productSupplier.name = params.sourceName ?: product?.name
```
Option B
```
productSupplier.name = params.sourceName ?: null
```
I believe Kelsey mentioned that she would prefer Option B, but let's ask her.

**Review by jmiranda — CHANGES_REQUESTED** [review]

File: grails-app/services/org/pih/warehouse/data/ProductSupplierDataService.groovy (hunk 1 above)
Comment (jmiranda) [reply grails-app/services/org/pih/warehouse/data/ProductSupplierDataService.groovy:144]: Kelsey says use product.name so let's go with Option A.

**Comment by DominikaHatala** [comment]
@jmiranda I have updated this PR with requested changes

**Comment by awalkowiak** [comment]
Looks like everything is fine now. I am merging this.

**Pull request merged**